src/include/daos_types.h

```diff
@@ -879,6 +879,15 @@ struct d_tgt_list {
 	uint32_t	tl_nr;
 };
 
+#define TGTS_IGNORE		((d_rank_t)-1)
+/** to identify each obj shard's target */
+struct daos_shard_tgt {
+	uint32_t	st_rank;	/* rank of the shard */
+	uint32_t	st_shard;	/* shard index */
+	uint32_t	st_tgt_idx;	/* target xstream index */
+	uint32_t	st_pad;		/* padding */
+};
+
```

Comment:
I think daos_types.h is supposed to be exported to application, this should be in a header in src/included/daos_srv/

Reply:
Yes, I hope some management api will use this structure to identify the target or rank, but I did not have time to cleanup the structure yet. Anyway I think this structure should belong to part of the user interface? no?

Reply:
hmm, maybe I should move this to other place for this patch.

Reply:
I moved this to daos/object.h